### ISSUE TITLE
Feedback: selection of pizza should be first step

### ISSUE THREAD

**Issue by bruderol** [adiherzog/pizza-delivery#3]
Currently sitting in the "spec by ex"-workshop, I demostrated your app, because it is soooo cool (:+1:) and we are working on a similar webshop example for our topic. I realized that something is very unusal compared to other shops, and I recommend to improve the example accordingly: 

usualy the user wants to see what he can order in a webshop (or pizza delivery shop) before he enters customer information. Therefore i recommend to display the "choose your pizza" step first and after the "choose your drink" step and only then ask for the phone number and then the address. 

Or is there a reason why the choose phone number is displayed first?

**Comment by adiherzog** [comment]
Yeah that would be more realistic. Maybe I will change it some day. But for my use case its more practical like this. Because I can show one important point: Not every scenario needs to test the entire use case, some scenarios can also just test a certain part of a scenario. I currently have one scenario that tests the entire process from start to end and two short scenarios that just test the phone number / address part. If I would move this part to the end, those two tests would also have to click through the entire flow and the difference to the main scenario would be less obvious.

**Comment by bruderol** [comment]
aha, okay, interessting.
Dont worry my feedback was more a "long term" feedback, it is not important for our presentation.

I will probably just change the example (on my own fork) for my own purpose accordingly (for the Spec-by-Example-Group).
